test/boost/mutation_reader_test.cc

```diff
@@ -2731,14 +2731,15 @@ SEASTAR_THREAD_TEST_CASE(test_manual_paused_evictable_reader_is_mutation_source)
     public:
         maybe_pausing_reader(
                 memtable& mt,
+                schema_ptr schema,
```

Comment:
Perhaps `query_schema` would be a better name

Reply:
Fair enough, this is a new variable, it might as well have the unambiguous name.